py/torch_tensorrt/runtime/_cudagraphs.py

```diff
@@ -74,7 +74,15 @@ def __enter__(self) -> torch.nn.Module:
 
         num_torch_module = 0
         num_trt_module = 0
-        for name, _ in self.compiled_module.named_children():
+        for name, module in self.compiled_module.named_children():
+            # need to disable cudagraphs if any model requires output allocator
+            if (
+                hasattr(module, "requires_output_allocator")
+                and module.requires_output_allocator
+            ):
+                raise RuntimeError(
+                    "There are converters that require Output Allocator. Please disable CUDA Graphs."
```

Comment:
There are submodules in this module which require the dynamic output allocator runtime mode which is incompatible with CUDA Graphs, please disable CUDA Graphs.